utils/hashPassword.js

```diff
@@ -0,0 +1,7 @@
+const bcrypt = require('bcrypt');
+
+const saltRounds = 10;
+
+const hashPassword = password => bcrypt.hashSync(password, saltRounds);
```

Comment:
Maybe `hashString` would be more generic. And remember the file names in `spine-case.js`

Reply:
Done!